src/extra/normalize.css

```diff
@@ -37,6 +37,21 @@
   outline-offset: 5px;
 }
 
+:where(a, button, input, textarea, summary) {
+  --outline-size: max(5px, 1px);
```

Comment:
can you explain this a bit? how'd you come across it?

Reply:
added some markup at docsite directory for testing from my mac

Reply:
just add some local custom variables for easier to understand